linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_unique_name/rule.yml

```diff
@@ -2,9 +2,15 @@ documentation_complete: true
 
 title: 'Ensure All Accounts on the System Have Unique Names'
 
-description: 'Change usernames, or delete accounts, so each has a unique name.'
+description: |-
+    Although the <tt>useradd</tt> utility prevents creation of duplicate user
+    names, it is possible for a malicious administrator to manually edit the
+    <tt>/etc/passwd</tt> file and change the user name.
 
-rationale: 'Unique usernames allow for accountability on the system.'
+rationale: |-
+    If a user is assigned a duplicate user name, the new user will be able to
+    create and have access to files with the first UID for that username as
+    defined in <tt>/etc/passwd</tt>.
```

Comment:
The original had a good rationale as well and referenced the auditability problem of having non-unique user names i.e. it's kinda hard when reviewing audit logs to know which user did what when there are 2 with the same user name.